data-mongodb/src/test/resources/logback.xml

```diff
@@ -9,7 +9,7 @@
         </encoder>
     </appender>
 
-    <root level="warn">
+    <root level="debug">
```

Comment:
Probably unwanted

Reply:
Yes, but I wanted also to see if there would be some failures on CI but will remove it before merging.

Reply:
Removed